R/bundlePackage.R

```diff
@@ -114,6 +114,6 @@ availablePackages <- function(repos) {
 }
 
 package_record <- function(name, lib_dir = NULL) {
-  path <- system.file("DESCRIPTION", package = name, lib.loc = lib_dir)
-  as.list(as.data.frame(read.dcf(path)))
+  record <- packageDescription(name, lib.loc = lib_dir, encoding = "UTF-8")
+  lapply(record, identity)
```

Comment:
Could that just be `as.list(record)`?

Reply:
That gives the error `Error: No method asJSON S3 class: packageDescription`. Not sure if there's a better way around this.